src/spatial/motion-dense.hpp

```diff
@@ -58,6 +58,15 @@ namespace pinocchio
       
       return X;
     }
+
+    HomogeneousMatrixType toHomogeneousMatrix_impl() const
+    {
+      HomogeneousMatrixType M;
+      M.template block<3,3>(0, 0) = skew(angular());
```

Comment:
I chose not to use `ANGULAR` and `LINEAR` here, as it would have been confusing to read `M(LINEAR, LINEAR) = angular` :thinking:

Since we already know the dimensions of the object we are working on anyway it would have been unnecessary concept-wrapping in my opinion.